src/HDAA_fmt_plug.c

```diff
@@ -68,6 +68,9 @@ john_register_one(&fmt_HDAA);
 // This is 8 x 64 bytes, so in MMX/SSE2 we support up to 9 limbs of MD5
 #define HTMP                    512
 
+// That's arbitrary because uri part is not limited by anything natural.
+#define MAX_CIPHERTEXT_LEN      8096
```

Comment:
8096 is a weird choice for an arbitrary limit - perhaps you meant 4096 or 8192?

Reply:
Ouch, I meant 8192 indeed.